NEWS

```diff
@@ -11,6 +11,15 @@
 ### Features:
 ### Bugfixes:
 
+## 1.15.0-rc2 (August 9, 2023)
+### Bugfixes:
+#### UCP
+* Fixed endpoint reconfiguration issues because of assymetrical selection
+#### UCM
+* Fixed conditional jump patching
+#### Tests
+* Multiple fixes for Azure CI pipeline
```

Comment:
maybe no need to mention as this is just internal infra

Reply:
removed